release/_publishing.sh

```diff
@@ -385,6 +385,8 @@ function _update_bundles_yml {
 		yq --indent 4 --inplace eval ".\"${product_version_key}\".\"${_PRODUCT_VERSION}\".latest = true" "${_PROJECTS_DIR}/liferay-docker/bundles.yml"
 	fi
 
+	sed -i "/^$/d" "${_PROJECTS_DIR}/liferay-docker/bundles.yml"
+	sed -i "s/$/\r/" "${_PROJECTS_DIR}/liferay-docker/bundles.yml"
```

Comment:
@kiwm, check if we really need this code after manually change the end of line character.